examples/starwars/pubspec.yaml

```diff
@@ -8,15 +8,14 @@ dependencies:
     path: ../../packages/graphql_flutter
   graphql:
     path: ../../packages/graphql
-  universal_platform: ^0.1.3
```

Comment:
What's the difference between these definitions?

Reply:
@RodolfoSilva just autoformatting from vscode, prob caused by the comment